plugins/woocommerce/i18n/states.php

```diff
@@ -831,17 +829,19 @@
 		'TN' => __( 'Tamil Nadu', 'woocommerce' ),
 		'TS' => __( 'Telangana', 'woocommerce' ),
 		'TR' => __( 'Tripura', 'woocommerce' ),
-		'UK' => __( 'Uttarakhand', 'woocommerce' ),
 		'UP' => __( 'Uttar Pradesh', 'woocommerce' ),
+		'UK' => __( 'Uttarakhand', 'woocommerce' ),
 		'WB' => __( 'West Bengal', 'woocommerce' ),
 		'AN' => __( 'Andaman and Nicobar Islands', 'woocommerce' ),
 		'CH' => __( 'Chandigarh', 'woocommerce' ),
 		'DN' => __( 'Dadra and Nagar Haveli', 'woocommerce' ),
-		'DD' => __( 'Daman and Diu', 'woocommerce' ),
+		'DD' => __( 'Daman and Diu', 'woocommerce' ),         
```

Comment:
What about [this](https://github.com/unicode-org/cldr/blob/2ef784e3a4168bc2a43cd1b5b9839b6636f5899c/common/subdivisions/en.xml#L2334)?
```suggestion
		'DD' => __( 'Daman and Diu', 'woocommerce' ),
		'DH' => __( 'Dādra and Nagar Haveli and Damān and Diu', 'woocommerce' ),
```